File: benches/baselines/micro/fib.lua
-- fib(30), recursive
local function fib(n)
  if n < 2 then return n end
  return fib(n - 1) + fib(n - 2)
end

local t0 = os.clock()
local r = fib(30)
local t1 = os.clock()
print(string.format("RESULT %d MS %d", r, math.floor((t1 - t0) * 1000)))

File: benches/baselines/micro/fib.js
// fib(30), recursive
function fib(n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

const t0 = Date.now();
const r = fib(30);
const t1 = Date.now();
print_line(`RESULT ${r} MS ${t1 - t0}`);
